bills-app/src/main/webapp/index.html

```diff
@@ -5,14 +5,19 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="api.js"></script> 
   </head>
-  <body>
+  <body onload = "addCode()">
```

Comment:
One more nit, but I'm gonna merge anyway--just wanted to point it out.
```suggestion
  <body onload="addCode()">
```